draft-ietf-quic-transport.md

```diff
@@ -3018,11 +3018,19 @@ elements on the network path, an endpoint verifies the following when an ACK
 frame is received:
 
 * The increase in ECT(0) and ECT(1) counters MUST be at least the number of QUIC
-  packets newly acknowledged that were sent with the corresponding codepoint.
+  packets newly acknowledged that were sent with the corresponding codepoint
+  minus the increase in the CE counter. This detects network remarking between
+  ECT(0) and ECT(1).
 
 * The total increase in ECT(0), ECT(1), and CE counters reported in the ACK
   frame MUST be at least the total number of QUIC packets newly acknowledged in
-  this ACK frame.
+  this ACK frame. This detects if the network changes ECT(0), ECT(1) or CE to
+  Not-ECT.
+
+If the sender does not have state to determine if a particular packet number is
+newly acknowledged or not, then the comparison SHOULD NOT be performed. By the
```

Comment:
Is this really a SHOULD NOT? If the sender doesn't have that state, it **CANNOT** perform that check. That's not a 2119 keyword, unfortunately :-)